components/desktop/gnome2/gnome-media/Makefile

```diff
@@ -64,4 +65,4 @@ test:		$(NO_TESTS)
 
 BUILD_PKG_DEPENDENCIES = $(BUILD_TOOLS)
 
-include $(WS_TOP)/make-rules/depend.mk
```

Comment:
It was a part of outdated logic to facilitate finding required packages, which was once used in userland-gate, but was never used in oi-userland (and was replaced by REQUIRED_PACKAGES rules). So, we remove these lines and eventually will remove make-rules/depend.mk.